.github/workflows/build.yml

```diff
@@ -19,9 +19,32 @@ jobs:
         with:
           java-version: '21.0.1'
           distribution: 'graalvm' # See 'Options' for all available distributions
+          cache: 'maven'
           github-token: ${{ secrets.GITHUB_TOKEN }}
       - name: Build and Test
         run: >
-          mvn -B clean package -Pjar
+          mvn -B clean verify -Pdependency-check
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
\ No newline at end of file
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # Needed to get PR information, if any
```

Comment:
only required for SonarCloud

Reply:
removed in https://github.com/cryptomator/hub-cli/pull/5/commits/370686fd2594bf8b97753bca28b90c616df79140